helm-charts/literacy-data-api/templates/deployment.yaml

```diff
@@ -0,0 +1,57 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "literacy-data-api.fullname" . }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "literacy-data-api.labels" . | nindent 4 }}
+spec:
+  {{- if not .Values.autoscaling.enabled }}
+  replicas: {{ .Values.replicaCount }}
+  {{- end }}
+  selector:
+    matchLabels:
+      {{- include "literacy-data-api.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      labels:
+        {{- include "literacy-data-api.selectorLabels" . | nindent 8 }}
+    spec:
+      volumes:
+        {{- range .Values.volumes }}
+        - name: {{ .name }}
+          secret:
+            secretName: {{ .secret.secretName }}
+        {{- end }}
+      containers:
+        - name: {{ .Chart.Name }}
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          env:
+            {{- range .Values.image.env }}
+            - name: {{ .name }}
+              value: {{ .value | quote}}
+            {{- end}}
+          volumeMounts:
+            {{- range .Values.image.volumeMounts }}
```

Comment:
Hrm, it's not really the _image_ that has the `volumeMounts` and the `envs`, it's the container (`image` and `imagePullPolicy` are features of the container, env/volumeMounts are other features of the container) -- so I wouldn't nest these under `image`, unless there's a good reason to?

Reply:
I moved these values to a an object named `container`